### ISSUE TITLE
Not working on few devices.

### ISSUE THREAD

**Issue by vikskod** [grantland/android-autofittextview#74]
I am using this library. It working fine on lots of devices but in few devices it do not work perfectly.
I am sharing screenshot between working fine (on Nexus 5) and not working (on Micromax Bold 2).
![screen shot 2016-08-15 at 1 31 16 pm](https://cloud.githubusercontent.com/assets/7125672/17658855/338724ba-62ee-11e6-8e3d-78fd05c427c4.png)

**Comment by amicloud** [comment]
Is this using an AutofitTextView, or is it using AutofitHelper.create()?

**Comment by vikskod** [comment]
Yes i am using AutofitTextView on Fragment which is attached to view pager.
